packages/webpack/test/fixtures/main/main.js

```diff
@@ -1,4 +1,5 @@
 import { hello as helloCommonJS } from 'commonjs-package'
+import { re } from 'commonjs-quirks'
```

Comment:
what's this for?

Reply:
To assert that we not only don't throw on the weird behavior of semver pkg, but also properly accept its export. I kept the original name for no reason other than it being searchable in node_modules